### PR TITLE
Fixed .form-control-select with caret issue

### PR DESCRIPTION
**Pull request by ashenden**

### CODE REVIEW

File: styles/components/form/form-control/shapes/form-control-jumbo/variables.less [hunk 1]
```diff
@@ -1,162 +1,162 @@
...
-@form-control-jumbo-margin-right: null;
-@form-control-jumbo-margin-right-screen-small: null;
-@form-control-jumbo-margin-right-screen-medium: null;
-@form-control-jumbo-margin-right-screen-large: null;
-@form-control-jumbo-margin-right-screen-jumbo: null;
+@form-control-jumbo-margin-right:                                               null;
+@form-control-jumbo-margin-right-screen-small:                                  null;
+@form-control-jumbo-margin-right-screen-medium:                                 null;
+@form-control-jumbo-margin-right-screen-large:                                  null;
+@form-control-jumbo-margin-right-screen-jumbo:                                  null;
```
Comment (jfurrow) [inline styles/components/form/form-control/shapes/form-control-jumbo/variables.less]: This extra spacing snuck in here

**Review by jfurrow — CHANGES_REQUESTED** [review]
Aside from the extra whitespace characters, this LGTM!

File: styles/components/form/form-control/shapes/form-control-jumbo/variables.less (hunk 1 above)
Comment (ashenden) [reply styles/components/form/form-control/shapes/form-control-jumbo/variables.less]: Strange.  Don't know how that happened, given I rebased from `canvas-v1` which already had the spaces removed.  Anyways.  Fixed.

**Review by ashenden — COMMENTED** [review]
See comments inline.  Fixed.

**Comment by jfurrow** [comment]
🚢

**Pull request merged**